tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -1851,4 +1851,8 @@
     <value>Loading adaptive card from the {0} provider</value>
     <comment>Text to tell the user that Dev Home is loading the adaptive card content from a specific Dev Home extension provider. {0} is the display name of the provider</comment>
   </data>
+  <data name="RepoToolNoRepositoriesMessage" xml:space="preserve">
+    <value>Added repositories are found in your {0} account and appear here.</value>

```

Comment:
>Added repositories are found in your {0} account and appear here. [](http://example.com/codeflow?start=11&length=65)

Why isn't it simply "There are no repositories in your {0} account."?

Reply:
Content peeps wanted this.  They also wanted to let users know where DevHome is pulling this information from.